styles/components/ItemFilters.module.scss

```diff
@@ -1,30 +1,44 @@
 @import "../utils/mixins";
 
-.itemFilterOptionsContainer {
-  position: absolute;
-  word-wrap: break-word;
-  z-index: 1;
-  background-color: white;
-  width: 230px;
-  padding: 10px;
-  border: 1px solid var(--nypl-colors-ui-gray-medium);
-}
-
+// The gray box that holds the filter groups
 .filtersContainer {
+  display: flex;
+  flex-wrap: wrap;
   background-color: var(--nypl-colors-ui-gray-x-light-cool);
   padding: 16px;
   min-height: 128px;
   z-index: 0;
+  gap: var(--nypl-space-s);
 }
 
-.filterGroup {
-  display: flex;
-  width: fit-content;
+// filter group is a generic class for any filters that may exist. As of 11/22
+// there are the aggregation filters and the search by year filter
+.filterGroupMobile {
+  flex-grow: 1;
```

Comment:
I can't figure out why but between 500-600px, both containers want to stay in the same row. You can add `width: 100%` here so that the filter container and year search container span the full width. Otherwise, this happens:
<img width="537" alt="Screen Shot 2023-11-27 at 4 22 17 PM" src="https://github.com/NYPL/research-catalog/assets/1280564/57b92400-cb2e-4cc2-8f5b-8df620cc021f">